server/db/models/Gig.js

```diff
@@ -10,6 +10,49 @@ module.exports = function (sequelize, DataTypes) {
   }, {
     classMethods: {
 
+      getGigs: function (organizationId) {
+        return Gig.findAll({
+          where: {
+            organizationId: organizationId
+          }
+        });
+      },
+
+      createGig: function (gigParams) {
+        return Gig.create(gigParams);
+      },
+
+      getGigInfo: function (id) {
+        return Gig.find({
+          where: {
+            id: id
+          }
+        });
+      },
+
+      updateGigInfo: function (id, params) {
+        return Gig.update(params, {
+          where: {
+            id: id
+          }
+        });
+      },
+
+      deleteGig: function (id) {
+        return Gig.find({
```

Comment:
I dont think it works like that: http://stackoverflow.com/questions/8402597/sequelize-js-delete-query

I could delete it in one line with a raw SQL theory. id rather leave it as it is now and come back and optimize it later